### ISSUE TITLE
Autogenerated relation map

### ISSUE THREAD

**Issue by quesada** [zadam/trilium#521]
One feature that is truly wonderful about Trilium is the relation map notetype.
I come from zim (Trilium beats it in every possible way!) One feature I used there is a 'link map', which shows a graph on how notes are connected to each other via note links (ctrl L). That is, you go about your day normally, linking notes, and whenever you want to see their relations, you click 'link map' and have an overview. It's really useful.

The way I'm using trilium, the relation maps are built 'by hand', they are very intentional, and they display relations in tags, not in internal linking. 

As wonderful a thinking tool as this is, I think the internal linking visualization would be really useful. You already have the library to render the nodes and links, maybe it'd be not that difficult?
I've read https://github.com/zadam/trilium/wiki/Patterns-of-personal-knowledge-base , so I see the way you use a notetaker doesn't depent all that much on internote linking. Just dropping this idea here as it may not have occurred to you.

**Comment by zadam** [comment]
Hello, this is an interesting topic which touches some of my plans.

Links are already being parsed out of the text and stored in a similar format as relations, essentially as a preparation for some kind link visualization. In the beginning of relation map implementation I experimented with auto-layouting the notes and relations, but the results were not satisfactory enough (for more complex graphs) so I opted to implement relation maps as manually positioned.

I do have plans to visualize note's outgoing and incoming relationship and it looks quite similar to zim link map. I'm adding this to backlog then.
